CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - `[jest-cli]` Break dependency cycle when using Jest programmatically ([#7707](https://github.com/facebook/jest/pull/7707))
 - `[jest-config]` Extract setupFilesAfterEnv from preset ([#7724](https://github.com/facebook/jest/pull/7724))
+- `[jest-cli]` Fail tests if logging happens after test environment is torn down ([#7731](https://github.com/facebook/jest/pull/7731))
```

Comment:
can you change the message (and PR name maybe) since we're not yet exit with 1?

Reply:
good call, done